src/components/ContactsList/ContactsListItem.vue

```diff
@@ -62,18 +64,28 @@ export default {
 			return this.contact.dav && (this.contact.dav.hasphoto || this.contact.photo)
 		},
 
-		/**
-		 * avatar color based on server toRgb method and the displayName
-		 * @returns {string} the color in css format
-		 */
-		colorAvatar() {
+		avatarStyle() {
+			if (this.hasPhoto) {
+				return {
+					backgroundColor: '#fff',
+					// The contact photo gets cropped in a circular shape, which might look odd with transparent photos.
+					// This box shadow ensures that there's always a very faint edge hinting at the circle.
+					boxShadow: '0 0 6px rgba(0, 0, 0, 0.08) inset',
```

Comment:
```suggestion
					boxShadow: '0 0 5px rgba(0, 0, 0, 0.05) inset',
```
Let’s go for a bit more subdued here. :)

Reply:
For reference, this is what Twitter uses and looks quite nice:
```
box-shadow: rgba(0, 0, 0, 0.02) 0px 0px 2px inset;
border: 1px solid rgba(0, 0, 0, 0.04);
```

Reply:
Also, we have a variable for this, no?